### PR TITLE
ksvgtopng5: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/ksvgtopng5.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ksvgtopng5
+
+> Ksvgtopng5: Convert SVG files to PNG format.
```
Comment (agnivade) [inline pages/linux/ksvgtopng5.md:3]: There's no need to repeat the command name again in the heading.

Also, is there a "more information" link that you can add?

File: pages/linux/ksvgtopng5.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- Convert a SVG file to PNG:
```
Comment (agnivade) [inline pages/linux/ksvgtopng5.md:5]: an

File: pages/linux/ksvgtopng5.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+`ksvgtopng5 {{width}} {{height}} {{full_path/to/file}} {{output_filename}}`
```
Comment (agnivade) [inline pages/linux/ksvgtopng5.md:7]: path/to/file is our usual convention that we use. It indicates both full and relative file paths.

Also, does the output file need to have a ".png" extension? Maybe good to write the example as path/to/input.svg and output.png.

**Review by agnivade — CHANGES_REQUESTED** [review]
Thanks for the PR. Please take a look at the comments.

File: pages/linux/ksvgtopng5.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/ksvgtopng5.md:3]: No, I can't find a link to more resources.

File: pages/linux/ksvgtopng5.md (hunk 3 above)
Comment (navarroaxel) [reply pages/linux/ksvgtopng5.md:7]: the source file should have an absolute path

**Review by sbrl — APPROVED** [review]
Looks ok to me. Thanks, @navarroaxel!

File: pages/linux/ksvgtopng5.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# ksvgtopng5
+
+> Convert SVG files to PNG format.
+
+- Convert an SVG file (should be an absolute path) to PNG:
+
+`ksvgtopng5 {{width}} {{height}} {{full_path/to/file.svg}} {{output_filename.png}}`
```
Comment (agnivade) [inline pages/linux/ksvgtopng5.md:7]: I think we can mention the bit about full path at only one place. Either the description or the example. I am fine with either.

Comment (navarroaxel) [reply pages/linux/ksvgtopng5.md:7]: I put it in the description

Comment (agnivade) [reply pages/linux/ksvgtopng5.md:7]: Sure, can we then make the file path as "path/to/file.svg"?

Comment (navarroaxel) [reply pages/linux/ksvgtopng5.md]: done

**Review by agnivade — APPROVED** [review]
Thank you for bearing with the review process @navarroaxel! Looks great now.

**Review by zdroid — APPROVED** [review]

**Pull request merged**